tools/project_manager/page_detail.php

```diff
@@ -17,19 +17,23 @@
 //    <username> = show specific user
```

Comment:
if round is not valid, I get a fatal error. If we had the options, we would encounter this less often unless someone is manipulating query params.

Reply:
ACK. It's thrown early in the input processing by the call to `get_round_param` which is good, that's what we want. And now, as you say, users are less likely to trip over it unless they're "hacking" URLs.